uninstall.php

```diff
@@ -35,7 +35,8 @@
  *
  * @param bool $erase Whether to erase data. Default false.
  */
-$erase = (bool) apply_filters( 'web_stories_erase_data_on_uninstall', false );
+
+$erase = (bool) get_option( \Google\Web_Stories\Settings::SETTING_NAME_DATA_REMOVAL );
```

Comment:
@timarney Now the filter docblock is duplicated in this file. Please check again.

And my bad for changing the version number. It should of course stay `@since 1.0.0`

Reply:
Right -- missed that ... updated now.

Reply:
It's _still_ duplicated

![Screenshot 2022-07-19 at 12 48 03](https://user-images.githubusercontent.com/841956/179732936-a096a681-e68b-48bf-9866-0b8e13dfbd34.png)